app/http.service.ts

```diff
@@ -0,0 +1,15 @@
+import {Injectable} from '@angular/core';
+import {Http, Response} from '@angular/http';
+import 'rxjs/Rx';
+
+@Injectable()
+export class HttpService {
```

Comment:
@LeraKruchek make sense. Moved getCities method to geo.service.ts file and remove unnecessary http.service.